internal/armada/repository/job.go

```diff
@@ -144,6 +148,7 @@ func (repo *RedisJobRepository) AddJobs(jobs []*api.Job) ([]*SubmitJobResult, er
 
 		submitResult.saveJobResult = pipe.Set(jobObjectPrefix+job.Id, jobData, 0)
 		submitResult.jobSetIndexResult = pipe.SAdd(jobSetPrefix+job.JobSetId, job.Id)
+		submitResult.initJobRetriesResult = pipe.Set(jobRetriesPrefix+job.Id, 0, 0)
```

Comment:
You might not need this, just assume there were no retries when the key is missing.